content/css/concepts/selectors/selectors.md

```diff
@@ -50,7 +50,7 @@ p {
 
 ## Selecting by Attribute
 
-Most commonly, the `class` and `id` attributes are assigned to elements for styling purposes. An `id` cannot start with a number.
+The `class` and `id` attributes are generally assigned for styling different elements on the same page and linking to sections or other pages of the site.
```

Comment:
```suggestion
The `class` and `id` attributes are generally assigned for styling different elements. An `id` cannot start with a number.
```
`class` is not used for linking. And the linking use case is outside the use as selectors. Also deleted a relevant sentence.